ui/src/components/Sidebar/Sidebar.tsx

```diff
@@ -64,6 +66,15 @@ export function GroupsAppMenu() {
                 Submit Feedback
               </DropdownMenu.Item>
             </a>
+            <Link
+              to="/about"
+              className="no-underline"
+              state={{ backgroundLocation: location }}
+            >
+              <DropdownMenu.Item className="dropdown-item pl-3 text-blue">
```

Comment:
in the video it looked like this text was center aligned, but it should be left aligned

Reply:
The other dropdown item is center aligned, though.

Reply:
weird I didn't think we center aligned anything lol